### ISSUE TITLE
Can't add multiple classes

### ISSUE THREAD

**Issue by sydney-d** [JPeer264/toastr2#3]
Hello,

Thank you for this great fork.

If I instanciate toastr like this : 

```js
const toastr = new Toastr({
    toastClass: 'class1 class2',
})
```

I get this error : 
```
Uncaught DOMException: Failed to execute 'add' on 'DOMTokenList': The token provided ('class1 class2') contains HTML space characters, which are not valid in tokens.
```

(it works with the jquery version since `$.addClass()` accepts space separated classes).

You could do : 
`$toastElement.classList.add(...options.toastClass.split(' '), iconClass);`

Could be nice to have this behaviour for position and icon classes too.

Best,

**Comment by JPeer264** [comment]
Thanks for your issue. I will try to fix this asap :)

**Comment by JPeer264** [comment]
@sydney-d should be fixed in v`3.0.0-alpha.9`
